DQM/DTMonitorClient/src/DTBlockedROChannelsTest.cc

```diff
@@ -59,8 +59,9 @@ void DTBlockedROChannelsTest::fillChamberMap( DQMStore::IGetter & igetter, const
   // get the RO mapping
   context.get<DTReadOutMappingRcd>().get(mapping);
 
-  // fill the map of the robs per chamber
-  for(int dduId = FEDNumbering::MINDTFEDID; dduId<=FEDNumbering::MAXDTFEDID; ++dduId) { //loop over DDUs
+  // fill the map of the robs per chamber 
+  // //FIXME: monitoring only real used FEDs
+  for(int dduId = FEDNumbering::MINDTFEDID; dduId<=774; ++dduId) { //loop over DDUs
```

Comment:
sorry for my very slow answer - so this means your offline monitoring will break (with no fast solution) in case these other feds are used. Perhaps MAXDTFEDID should be changed (in consultation with daq)  if this is considered a permanent solution for DT